addons/binding/org.openhab.binding.airvisualnode/src/main/java/org/openhab/binding/airvisualnode/AirVisualNodeBindingConstants.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.airvisualnode;
+
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+import com.google.common.collect.ImmutableSet;
+
+/**
+ * The {@link AirVisualNodeBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Victor Antonovich - Initial contribution
+ */
+public class AirVisualNodeBindingConstants {
+
+    @NonNull public static final String BINDING_ID = "airvisualnode";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_AVNODE = new ThingTypeUID(BINDING_ID, "avnode");
+
+    // List of all Channel ids
+    public static final String CHANNEL_CO2_PPM = "co2_ppm";
+    public static final String CHANNEL_HUMIDITY = "humidity";
+    public static final String CHANNEL_AQI_CN = "aqi_cn";
+    public static final String CHANNEL_AQI_US = "aqi_us";
```

Comment:
This is the same channel as the above one, but with a different unit, right? If you use UOM you only need one channel for this.